### PR TITLE
div. Bugs

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: app/grails-app/i18n/messages.properties [hunk 1]
```diff
@@ -413,7 +413,7 @@ default.compare.subscriptions = Select subscriptions to compare:
 default.compare.show.activeSubscriptions.name=active Subscriptions
 default.compare.show.intendedSubscriptions.name=intended Subscriptions
 default.compare.show.subscriber.name=Subscriber Subscriptions
-default.compare.show.conntectedSubscriptions.name=Conntected Subscriptions
+default.compare.show.conntectedObjects.name=Conntected Subscriptions
```
Comment (croqueGrec09) [inline app/grails-app/i18n/messages.properties:416]: conntected

File: app/grails-app/i18n/messages_de.properties [hunk 2]
```diff
@@ -410,7 +410,7 @@ default.compare.subscriptions = Zu vergleichende Lizenzen auswählen:
 default.compare.show.activeSubscriptions.name=aktive Lizenzen
 default.compare.show.intendedSubscriptions.name=geplante Lizenzen
 default.compare.show.subscriber.name=Teilnehmerlizenzen
-default.compare.show.conntectedSubscriptions.name=verknüpfte Lizenzen
+default.compare.show.conntectedObjects.name=verknüpfte Lizenzen
```
Comment (croqueGrec09) [inline app/grails-app/i18n/messages_de.properties:413]: conntected

File: app/grails-app/services/de/laser/SubscriptionService.groovy [hunk 3]
```diff
@@ -447,38 +447,38 @@ class SubscriptionService {
             tmpQ = getSubscriptionsLocalLicenseQuery(params)
             result.addAll(Subscription.executeQuery("select s " + tmpQ[0], tmpQ[1]))
         }
-        if (params?.showConnectedSubs){
+        if (params?.showConnectedObjs){
```
Comment (croqueGrec09) [inline app/grails-app/services/de/laser/SubscriptionService.groovy:450]: params?

**Review by croqueGrec09 — COMMENTED** [review]
remarks

**Review by croqueGrec09 — APPROVED** [review]
approved

**Review by croqueGrec09 — APPROVED** [review]
approved

**Review by klodav — APPROVED** [review]

**Pull request merged**